src/control/SConscript

```diff
@@ -148,60 +148,34 @@ def scons():
     drpcbin = get_bin_path(gopath, "drpc_test")
     AlwaysBuild([agentbin, dmgbin, adminbin, drpcbin])
 
+    SConscript('lib/spdk/SConscript', exports='denv')
+
     senv = denv.Clone()
     prereqs.require(senv, 'pmdk', 'spdk', 'ofi', 'hwloc')
 
-    gospdkpath = join(gosrc, "lib", "spdk")
-
-    # CGO prerequisite libs for go-spdk bindings
-    senv.AppendUnique(CPPPATH=[join(gospdkpath, "include")])
-
-    # Link to DPDK static libs
-    senv.AppendUnique(LINKFLAGS=['-Wl,--whole-archive', \
-                      '-lrte_mempool', '-lrte_mempool_ring', '-lrte_bus_pci', \
-                      '-lrte_pci', '-lrte_ring', '-lrte_mbuf', '-lrte_eal', \
-                      '-lrte_kvargs', \
-                      '-Wl,--no-whole-archive'])
-
-    # hack to avoid building this library with cov compiler for the moment
-    compiler = senv.get('COMPILER').lower()
-    if compiler == "covc":
-        compiler = "gcc"
-
-    nvmecontrol = senv.StaticLibrary(
-        "nvme_control",
-        join(gospdkpath, "src", "nvme_control.c"),
-        CC=compiler, LIBS=['spdk', 'spdk_env_dpdk', 'numa'])
-
-    senv.Install(join(senv.subst("$PREFIX"), "lib64"), nvmecontrol)
-
-    # CGO shell env vars.
-
-    # explicitly link RTE & SPDK libs
-    senv.AppendENVPath(
-        "CGO_LDFLAGS",
-        senv.subst("-L%s -L$SPDK_PREFIX/lib -L$HWLOC_PREFIX/lib"
-                   " -L$OFI_PREFIX/lib -L$ISAL_PREFIX/lib $_RPATH" % gopath) + \
-                   " -lspdk_env_dpdk -lspdk_thread -lspdk_bdev -lspdk_copy" + \
-                   " -lrte_mempool  -lrte_mempool_ring -lrte_bus_pci" + \
-                   " -lrte_pci -lrte_ring -lrte_mbuf -lrte_eal -lrte_kvargs" + \
-                   " -lspdk_bdev_aio -lspdk_bdev_nvme -lspdk_bdev_malloc" + \
-                   " -lspdk_conf -lspdk_blob -lspdk_nvme -lspdk_util" + \
-                   " -lspdk_json -lspdk_jsonrpc -lspdk_rpc -lspdk_trace" + \
-                   " -lspdk_sock -lspdk_log -lspdk_notify -lspdk_blob_bdev " + \
-                   " -lnuma -ldl -lisal")
-
-    senv.AppendENVPath(
-        "CGO_CFLAGS",
-        senv.subst("-I$SPDK_PREFIX/include -I$HWLOC_PREFIX/include"
-                   " -I$OFI_PREFIX/include"))
-
-    # copy setup_spdk.sh script to be executed at daos_server runtime
+    # Explicitly link RTE & SPDK libs for CGO access.
+    senv.AppendENVPath("CGO_LDFLAGS",
+        senv.subst("-L$PREFIX/lib64 -L$SPDK_PREFIX/lib -L$HWLOC_PREFIX/lib "
```

Comment:
(pylint-bad-continuation) Wrong continued indentation (add 15 spaces).

Reply:
It wants you to line up this line with “CGO...

It might be a little cleaner to construct this part outside the function call.  Perhaps in a loop.